File: problem0005.java
/**
 * Problem url: https://projecteuler.net/problem=5
 * 
 * @author Josep Monclus
 */
public class problem0005 {
    public static void main(String[] args) {
        int maxMult = 20;

        int value = 1;

        while(true) {
            boolean found = true;

            for(int i = 1; i <= maxMult; i++) {
                if(value % i != 0) {
                    found = false;
                    break;
                }
            }

            if(found) break;

            value++;
        }

        System.out.println("Value: " + value);
    }
}

File: python/problem_0005.py

# Problem url: https://projecteuler.net/problem=5

import time

start = time.time()

n = 1

while True:
    found = True
    for i in range(20, 1, -1):
        if n % i != 0:
            found = False
            break
    
    if found:
        print("Answer is: ", n)
        break
    else:
        n += 1

end = time.time()
print(end - start)